src/check/parse/tokenize.zig

```diff
@@ -696,12 +706,12 @@ pub const Tokenizer = struct {
                 .SingleQuoteInterpolation, .SingleQuoteInterpolationMultiline => '\'',
                 else => return,
             };
-            const kind: StringKind = switch (brace_kind) {
-                .StringInterpolationMultiline, .SingleQuoteInterpolationMultiline => .multi_line,
-                else => .single_line,
+            const multiline = switch (brace_kind) {
```

Comment:
Why remove the enum here?

Reply:
I this is me being behind the latest version you published.  I'll check it out and fix this up